Packs/RSTCloud/Playbooks/Domain_Enrichment_-_RST_Threat_Feed.yml

```diff
@@ -0,0 +1,243 @@
+id: Domain Enrichment - RST Threat Feed
+version: -1
+fromversion: 5.5.0
```

Comment:
Why is this fromversion `5.5.0` while the integration is 5.0.0?

Reply:
updated to be 5.0.0